### ISSUE TITLE
Correct accessibility label for dismissing menus

### ISSUE THREAD

**Issue by guidezpl** [flutter/flutter#118994]
The accessibility label for dismissing any menu should be "Dismiss context menu, Button", not "Dismiss, Button".

**Comment by HansMuller** [comment]
This should taken care of as part of https://github.com/flutter/flutter/issues/23600 (which is part of the Material 3 project: https://github.com/flutter/flutter/issues/91605).
